client/src/components/exercise/lab5/pages/FormResponseActivity/FormRepair.js

```diff
@@ -38,11 +38,11 @@ class FormRepair extends Component {
     Object.keys(this.state).map((name) => {
       switch (name) {
         case "errorNotification":
-          if (this.state[name] !== "Please enter in format: YYYY-MM-DD") {
+          if (this.state[name] !== '"Please enter in format: YYYY-MM-DD"') {
             error = true;
             this.setState({
               errorNotificationerror:
-                "Must enter 'Please enter in format: YYYY-MM-DD'",
```

Comment:
For all of these repairs, should we let users enter either single quotes or double quotes? 
Should `'Please enter in format: YYYY-MM-DD'` and `"Please enter in format: YYYY-MM-DD"` pass?
Not sure if there is a right answer, just something to think about

Reply:
I Agree with this change it makes the most sense because both work @notcruz